Show a checklist of required data on "Loading" screen

pablen opened the pull request:
<img width="457" alt="captura de pantalla 2018-07-23 a la s 17 30 50" src="https://user-images.githubusercontent.com/3164763/43103520-bf973178-8ea4-11e8-869d-37f9701b43e3.png">


Closes #399

Commit:
Show a checklist of required data on "Loading" screen

Closes #399

gabmontes reviewed (commented):
What if we move ahead a step further?

- Replace the "Contacting network" message by "Gathering information".
- Replace "Latest block" by "Blockchain status" (more user-friendly).
- Completely remove the token status as it is now (and for the rest of the eternity) `true`.
- Center de list. It looks like it is a bit to the left.

Commit:
Apply additional changes

- Replace the "Contacting network" message by "Gathering information".
- Replace "Latest block" by "Blockchain status" (more user-friendly).
- Completely remove the token status as it is now (and for the rest of the eternity) true.
- Center de list.
